src/main/java/de/thomas_oster/liblasercut/drivers/GenericGcodeDriver.java

```diff
@@ -549,8 +556,23 @@ protected void http_upload(URI url, String data, String filename) throws IOExcep
 
   protected void http_play(String filename) throws IOException, URISyntaxException
   {
+    String command = "play /sd/"+filename;
+    http_command(command);
+  }
+
+  protected void http_commands(String commands, String filename) throws IOException, URISyntaxException
+  {
+    for (String command : commands.split(","))
```

Comment:
Is , a good separator here? That prevents using a , inside the gcode commands sent? Ideally, you would just use a newline, but I think the GUI does not allow that?

Reply:
The reason for a comma as a separator is that this is also used with preJobGcode and postJobGcode

Reply:
Good point, better to use comma here for consistency as well, then.